### ISSUE TITLE
New click library version (8.1.0) breaks spaCy

### ISSUE THREAD

**Issue by WBrandes** [explosion/spaCy#10564]
## How to reproduce the behaviour
Trying to import spaCy in a python terminal, or trying to download a model through the command line (`python -m spacy download en_core_web_trf`), gives:

```
ImportError: cannot import name 'get_terminal_size' from 'click.termui
```

Only ran into this because I just made a new AWS instance, and it installed the newest version of click, which was updated about an hour ago (just my luck). When I re-installed click as version `8.0.4` spaCy runs fine, but I expect anyone doing a clean install of Python will run into this issue.

## Your Environment
* Operating System: Amazon Linux
* Python Version Used: 3.9.6
* spaCy Version Used: 3.2.3

**Comment by philgooch** [comment]
According to the click 8.1.0 Release Notes: `get_terminal_size is removed, use shutil.get_terminal_size instead.`

**Comment by AlexandrePieroux** [comment]
I resolved this by fixing the click version to the previous release: 8.0.4
It seems the entirely refactored the 'term.py' file and removed 'get_terminal_size' method...

**Comment by Mayank-Bhatia** [comment]
So what's the solution in the meantime, use older version of click after install spacy?

**Comment by Mayank-Bhatia** [comment]
`RUN pip uninstall click -y && pip install "click<8"` in my Dockerfile seems to do the trick.

**Comment by polm** [comment]
Thanks for the reports everyone! I would say for the time being please downgrade click. We'll work on having a patch out for this soon.

**Comment by philgooch** [comment]
Adding `click==8.0.4` at the end of my `requirements.txt` works for me. Alternatively, adding `pip install click==8.0.4` within the Dockerfile after installing other dependencies also works.

**Comment by svlandeg** [comment]
- Upstream issue: https://github.com/tiangolo/typer/issues/377
- Upstream fix (to be released): https://github.com/tiangolo/typer/pull/375

**Comment by svlandeg** [comment]
As the upstream issue is not fixed yet, we released [v3.2.4](https://github.com/explosion/spaCy/releases/tag/v3.2.4) to deal with this issue.

**Comment by tiangolo** [comment]
Hello! I just released Typer `0.4.1`, you can now upgrade the version and that should handle it. :rocket: :nerd_face:
